### ISSUE TITLE
Can I add some domains to lantern PAC

### ISSUE THREAD

**Issue by aelam** [getlantern/lantern-binaries#4]
Some domains are blocked by my company. And I don't want to proxy all my traffic. 
Is it possible to let users add some domains by themselves ?

**Comment by yuan1xin2ying3** [comment]
good

**Comment by myleshorton** [comment]
Lantern should automatically detect that sites are blocked. Is the auto-detection not working for you? Can you give an example of a site where it's not working @aelam or @yuan1xin2ying3?

**Comment by aelam** [comment]
Some websites are blocked by my company. I am not sure the strategy of it. 
It shows me a page like 'contact with administrator' 
Maybe the point is it doesn't response a 404 error .

**Comment by aelam** [comment]
![screen shot 2016-05-18 at 1 33 31 pm](https://cloud.githubusercontent.com/assets/232337/15348418/4abe8bb4-1cfd-11e6-9039-5db8d7b901d8.png)
![screen shot 2016-05-18 at 1 33 20 pm](https://cloud.githubusercontent.com/assets/232337/15348425/5544c30a-1cfd-11e6-84ad-f70f4cc40f51.png)

**Comment by softwareB** [comment]
win7-X86 which version is best？

**Comment by Artoria2e5** [comment]
It sounds technically possible to hand-edit the YAML files and add some sites:
```
proxiedsites:
  delta:
    additions: []
    deletions: []
```

**Comment by myleshorton** [comment]
Yeah cases like that are harder for Lantern to detect @aelam, but yes it would be fairly easy to add functionality to customize the list.

**Comment by keyingkai** [comment]
You can use https://pan.baidu.com @aelam

**Comment by Artoria2e5** [comment]
@keyingkai So the easiest solution would be "install HTTPS Everywhere"?
